CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 **New Feature:**
 - Add dynamic client [kubernetes-client/python-base#56](https://github.com/kubernetes-client/python-base/pull/56)
 
+**Breaking Change:**
+- The Python client will be generated by openapi-generator, with the following breaking changes [kubernetes-client/gen#97](https://github.com/kubernetes-client/gen/pull/97)
+- `kubernetes.client.apis` package is renamed to `kubernetes.client.api`
+- `kubernetes` package code now uses absolute import instead of relative import
+- API requests won't have `Content-Type` specified in HTTP headers
```

Comment:
would this not cause problem? we use json, or proto.

Reply:
I interpreted the change wrong. The code generator just omitted the content type if the request doesn't have a body. I removed this note from the changelog since it's not user-facing.